cli/cmd/project/tables.go

```diff
@@ -89,21 +91,19 @@ func TablesCmd(ch *cmdutil.Helper) *cobra.Command {
 
 				// Get row count using SQL query
 				var rowCount string
-				queryRes, err := rt.QueryServiceClient().Query(cmd.Context(), &runtimev1.QueryRequest{
-					InstanceId: instanceID,
-					Sql:        fmt.Sprintf("SELECT COUNT(*) FROM %s", drivers.DialectDuckDB.EscapeIdentifier(table.Name)),
+				countQuery := fmt.Sprintf("SELECT COUNT(*) FROM %s", drivers.DialectDuckDB.EscapeIdentifier(table.Name))
+				queryRes, err := rt.RuntimeServiceClient.QueryResolver(cmd.Context(), &runtimev1.QueryResolverRequest{
+					InstanceId:         instanceID,
+					Resolver:           "sql",
+					ResolverProperties: &structpb.Struct{Fields: map[string]*structpb.Value{"sql": {Kind: &structpb.Value_StringValue{StringValue: countQuery}}}},
```

Comment:
nit: use `structpb.NewStruct` instead of creating these deeply nested objects directly (you can combine it with a `must` util func to avoid handling the error)